Documentation/gettingstarted/http.rst

```diff
@@ -324,17 +335,36 @@ You can observe the L7 policy via ``kubectl``:
     Name:         rule1
     Namespace:    default
     Labels:       <none>
-    Annotations:  kubectl.kubernetes.io/last-applied-configuration:
-                    {"apiVersion":"cilium.io/v2","description":"L7 policy to restrict access to specific HTTP call","kind":"CiliumNetworkPolicy","metadata":{"...
-    API Version:  cilium.io/v2
+    Annotations:  API Version:  cilium.io/v2
```

Comment:
Is this correct? Previously `Annotations` and `API Version` were on separate lines.

Reply:
Good question :thinking: this is what I by from running `kubectl describe ciliumnetworkpolicies > cnp.txt` and then pasting the contents of `cnp.txt` here. Will check again.

Reply:
Just checked again on minikube. This is indeed what I get when running `kubectl describe ciliumnetworkpolicies`:

```
Name:         rule1
Namespace:    default
Labels:       <none>
Annotations:  API Version:  cilium.io/v2
Description:  L7 policy to restrict access to specific HTTP call
Kind:         CiliumNetworkPolicy
Metadata:
  Creation Timestamp:  2020-06-15T19:07:32Z
  Generation:          2
  Managed Fields:
    API Version:  cilium.io/v2
    Fields Type:  FieldsV1
    fieldsV1:
      f:description:
      f:metadata:
        f:annotations:
          .:
          f:kubectl.kubernetes.io/last-applied-configuration:
      f:spec:
        .:
        f:endpointSelector:
          .:
          f:matchLabels:
            .:
            f:class:
            f:org:
        f:ingress:
    Manager:         kubectl
    Operation:       Update
    Time:            2020-06-15T19:08:15Z
  Resource Version:  1012
  Self Link:         /apis/cilium.io/v2/namespaces/default/ciliumnetworkpolicies/rule1
  UID:               13e03438-c46c-4063-b74b-39240205557b
Spec:
  Endpoint Selector:
    Match Labels:
      Class:  deathstar
      Org:    empire
  Ingress:
    From Endpoints:
      Match Labels:
        Org:  empire
    To Ports:
      Ports:
        Port:      80
        Protocol:  TCP
      Rules:
        Http:
          Method:  POST
          Path:    /v1/request-landing
Events:            <none>
```

Reply:
*shrug*

If that's what it says, that's what it says. I see the same thing.